manual/manual/cmds/comp.etex

```diff
@@ -552,11 +552,22 @@ Display a short usage summary and exit.
 %
 \end{options}
 
-\noindent
-On native Windows, the following environment variable is also consulted:
+\paragraph{contextual control of command-line options}
+
+The compiler command line can be modified ``from the outside''
+with the following mechanisms. These are experimental
+and subject to change, and they are not intended for normal use.
```

Comment:
What about "and subject to change.  They should be used only for experimental and development work---never in released packages." ?